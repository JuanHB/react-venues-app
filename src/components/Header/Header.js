import React from 'react';

const Header = () => (
  <h1 className='h2'> Venues Near You </h1>
);

export default Header;
