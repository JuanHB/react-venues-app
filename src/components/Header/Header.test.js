import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

it('renders correctly header component', () => {
  const HeaderComponent = renderer.create(<Header />).toJSON();
  expect(HeaderComponent).toMatchSnapshot();
});
