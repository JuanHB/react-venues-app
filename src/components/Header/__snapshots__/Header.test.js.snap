// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly header component 1`] = `
<h1
  className="h2"
>
   Cool places near you 
</h1>
`;
